Libraries/LibGUI/GTabWidget.h

```diff
@@ -15,6 +15,7 @@ class GTabWidget : public GWidget {
 
     TabPosition tab_position() const { return m_tab_position; }
     void set_tab_position(TabPosition);
+    int get_active_tab();
```

Comment:
Please add this API in a separate commit, so it's isolated from the other changes.
Also, it should return be `const` and maybe have a slightly more descriptive name like:

```cpp
int active_tab_index() const;
```